test/run-openshift-remote-cluster

```diff
@@ -32,6 +28,10 @@ ct_os_check_compulsory_vars
 
 ct_os_check_login || exit 1
 
+ct_os_tag_image_for_cvp "python"
+
+ct_pull_or_import_postgresql
```

Comment:
I know I already gave LGTM, but I suppose we should add something like this here.
```suggestion
ct_pull_or_import_postgresql || exit 1
```